gluster-ansible.spec

```diff
@@ -1,14 +1,14 @@
 %global rolesdir %{_sysconfdir}/ansible/roles/gluster.ansible
 %global docdir %{_datadir}/doc/gluster.ansible
-%global buildnum 5
+%global buildnum 26
```

Comment:
it should be 27.

Reply:
Fixed in https://github.com/gluster/gluster-ansible/pull/137/commits/ee1728bc66295af57f673c0c6f9c352ac8186f04